src/data/faq.json

```diff
@@ -51,10 +51,13 @@
-                            "Remarks:<ul><li>Error reports are automatically deleted after 14 days.</li><li>Error reports that are transmitted to the server without request by the CWA team cannot be analyzed and processed.</li><li>There is no back channel into the Corona-Warn-App for the error reports. The CWA technical team can only communicate via GitHub to an issue.</li></ul>",
-                            "See also the blog article <a href='../blog/2021-05-12-corona-warn-app-version-2-2/' target='_blank' rel='noopener noreferrer'>Corona-Warn-App version 2.2 enables creation of a rapid test profile and error reports</a>."
+                            "With version 2.2, the Corona-Warn-App (CWA) offers a new function for detailed technical problem analysis. The error report is intended to help the developers to perform a root cause analysis faster and to eliminate errors more efficiently. <br><b>Important:</b> Please report an issue <b>first</b> to the technical hotline, either by phone (<a href='#international_phone_numbers' target='_blank' rel='noopener noreferrer'>click here to go see the phone numbers</a>) or by e-mail (<a href='mailto:corona-warn-app.opensource@sap.com'>corona-warn-app.opensource@sap.com</a>), to verify if there is already a known solution. An error report is only required if the technical support asks for this report.",
+                            "The best is to share the unique report ID where it was requested. You can also share the ID through the following ways:",
+                            "<ul><li>Open an issue on GitHub (<a href='https://github.com/corona-warn-app/cwa-app-android/issues/new/choose' target='_blank' rel='noopener noreferrer'>Android</a> or <a href='https://github.com/corona-warn-app/cwa-app-ios/issues/new/choose' target='_blank' rel='noopener noreferrer'>iOS</a>) and mention the ID of your error report</li><li> Share the ID with the technical support</li><li> Write a review on the Corona-Warn-App in the Apple App Store or Google Play Store and mention the ID of your error report</ul></li>",
+                            "Regardless of which contact option you choose, the following tips will help to resolve your issue as quickly as possible:",
+                            "<ol><li>Describe the problem as accurately as possible</li><li>Start the error analysis in the CWA (<a href='../blog/2021-05-12-corona-warn-app-version-2-2/' target='_blank' rel='noopener noreferrer'>see blog post</a>)<br>- Try to reproduce the incorrect behaviour or to generate the error message<br>- If this is successful, send the error report to the CWA server. The data will be anonymised and encrypted. If you are interested, you can also save the report locally on your smartphone<br>- You can then find your unique ID on the bottom of the 'Error Reports' screen, scroll down and click on 'ID History'<br>- Stop the error analysis</li><li>Share the 'unique report ID' with the technical support</li></ol>",

```

Comment:
> The data will be anonymised and encrypted

"anonymise" -> "anonymize" (small problem, but aligns the spelling with the rest of the entry)

Reply:
Done in https://github.com/corona-warn-app/cwa-website/pull/1267/commits/880a279dd6a7d5db22940a8e9fb58b8e745b9174